extension/content/firebug/firefox/browserMenu.js

```diff
@@ -436,15 +436,17 @@ var BrowserMenu =
     overlayFirefoxMenu: function(doc)
     {
         // Firefox page context menu
+      if (!this.browserOverlay.isMultiprocessEnabled() && !this.browserOverlay.auroraChannel) {
```

Comment:
The menu should be visible for users with e10s off, but why is the menu hidden for Aurora in this patch? 

nit: please fix the indentation (Firebug is using 4 spaces)

I've been testing with Aurora an the menu is always hidden no matter whether e10s is on or off. Btw. I removed the Aurora condition so, the code looked as follows:

```
if (this.browserOverlay.isMultiprocessEnabled()) {
}
``` 

The `isMultiprocessEnabled` function returns always false and need to be fixed.


My STR:
1) Launch Aurora
2) Go to Tools -> Options and uncheck "Enable multi-process Firefox Developer Edition", restart
3) Load e.g. google.com
4) Right click on the page and check that Firebug's inspect menu item is there. It isn't in my case -> BUG

Honza